### PR TITLE
 Update privacy reports UI for tablet (uplift to 1.13.x) 

### PR DESCRIPTION
**Pull request by deeppandya**
Uplift of #6563
Resolves https://github.com/brave/brave-browser/issues/11322

Approved, please ensure that before merging: 
- [x] You have checked CI and the builds, lint, and tests all pass or are not related to your PR. 
- [x] You have tested your change on Nightly. 
- [x] The PR milestones match the branch they are landing to. 

After you merge: 
- [x] The associated issue milestone is set to the smallest version that the changes is landed on.

### CODE REVIEW

**Comment by deeppandya** [comment]
CI is failing with `/home/ubuntu/workspace/pr-brave-browser-pr6563_privacy_report_update_tablet_1.13.x-android@tmp/durable-4982a883/script.sh: /home/ubuntu/android-sdk/tools/mksdcard: not found`

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.13.x` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-core/pull/6563#issuecomment-686279663.

**Pull request merged**
